Fixed Malden outpost_12 (Bad mission spawner)

Preen1 opened the pull request:
## What type of PR is this.
1. [x] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Updated the location of outpost_12, Outpost now supports ammo missions.

### Please specify which Issue this PR Resolves.
closes #1636 

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [x] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?
### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: Start the mission and run the following as local:
```[["outpost_12"],"A3A_fnc_LOG_Ammo"] remoteExec ["A3A_fnc_scheduler",2]```

********************************************************
Notes:
![20210204181936_1](https://user-images.githubusercontent.com/78278253/106972062-be52be80-6715-11eb-881c-d157b6d018e0.jpg)
![20210204181947_1](https://user-images.githubusercontent.com/78278253/106972066-beeb5500-6715-11eb-8f22-0a5ef3aaa9e9.jpg)

Need to verify the quality of outpost. @Bob-Murphy 
Check that the props used do not require DLC ownership ("A3_Props_F_Exp")

Bob-Murphy commented:
So you added a new outpost somewhere else on the map and deleted the one o lay because it couldn't support certain types of missions, right?

Please note that not every outpost/marker needs to be able to support every type of mission. It is one thing adding a new outpost to a map, but just taking one out and then leaving a blind spot on the map which happens to be close to the highest position on the map and a good strategic position is another one.

I have no clue where you have created the new outpost and I currently won't look at it because no time, but I can tell you that I don't really like the fact of the empty spot.

Nevertheless, you'll receive full feedback in the soon.

Lazejun commented:
Assuming by the Title you didn't fully read the Issue 😎

I mentioned the main Issue first, then additional stuff

Props don't and can't have DLC dependencies, the only people that might not be able to use Maps with for Example Contact stuff are those that Pirate the Game/use older versions

Lazejun reviewed (changes requested):
Half of the Issue was completly Ignored,

Radio tower is still way to close to the Helipad, in General its pretty tight for Ai to Land there.
![grafik](https://user-images.githubusercontent.com/57111907/107122744-cf2c3d00-6899-11eb-981e-da253f7ac584.png)


Helipad is on an Uneven surface, not Facing down the Hill
![20210206163946_1](https://user-images.githubusercontent.com/57111907/107122773-f256ec80-6899-11eb-99c3-6725bc829afd.jpg)

And this Piece of wall with a Hole is way to high.
![grafik](https://user-images.githubusercontent.com/57111907/107123630-d6a21500-689e-11eb-9576-642a51d588c6.png)


In Terms of Use of the Radiotower, it only Influeces 1 Town and half the Jaming radius is occupied by nearby other Radio towers.

![20210206170213_1](https://user-images.githubusercontent.com/57111907/107123332-3ac3d980-689d-11eb-8fa3-551db68ae3ad.jpg)

The Previous Pos of Outpost_12 Influenced 3 Towns.

![20210206170218_1](https://user-images.githubusercontent.com/57111907/107123366-6d6dd200-689d-11eb-9148-9d5ec8c2b228.jpg)

![20210206170715_1](https://user-images.githubusercontent.com/57111907/107123442-d81f0d80-689d-11eb-91a3-a9857093434a.jpg)

In General there was not much to gain by Moving Outpost_12, you could have added another Outpost in the abandoned Military Base if you think the North needs more stuff.

Preen1 commented:
outpost_12 VERSION 2:
The latest Change is a redesign of the outpost to fix the original issue of the tower being too close to the helipad. Also fixed the issue of the helipad being at an angle and fixed the broken walls to be crawling height instead of crouching. --Picture below
![20210207140117_1](https://user-images.githubusercontent.com/78278253/107159534-4d4e1980-6956-11eb-90a8-7cc048efb73b.jpg)

The physical location of the outpost did not substantially change. (Was moved ~15m down the hill to make the helipad level) 
Some more reorganization will likely be required if the outpost changes location.

Bob-Murphy commented:
Ok. Now I get what you did.
You moved the complete outpost from the position on top of the big mountain(red marker below)
![image](https://user-images.githubusercontent.com/50651578/107308252-97371c80-6a88-11eb-8c23-14c14644dbd5.png)
which had the issue of not being close to a road to another smaller hill further north(red marker below), which still has no road directly next to it to be able to spawn ammotruck missions.
![image](https://user-images.githubusercontent.com/50651578/107308753-7cb17300-6a89-11eb-8508-120fa3589273.png)

Based on the issue the only thing that needed to be done was the repositioning of the helipad in the old location.

Therefore:
Please reintroduce the outpost at the old location, of course you are free to take the overhauled design from the northern outpost as I don't think that such a large area (marked in blue) should be without any content.
We are accepting the fact that the outpost can not generate ammotruck missions.
![image](https://user-images.githubusercontent.com/50651578/107308723-6d322a00-6a89-11eb-8b3a-e5a4ac5a0735.png)

Also, please remove the radio tower in the northern position you placed for the reason Lazejun mentioned.

Bob-Murphy reviewed (changes requested):
- add back the outpost on the southern hill WITH the radio tower and make sure that the helipad is nicely placed
- delete the radiotower in the northern outpost you placed. The outpost in itself can stay.

Preen1 commented:
Revision 3: Outpost_12 has been moved back to its original location as per Bob's request. The outpost was redesigned slightly but still includes a radiotower, a functioning helipad, and a few garrisoned buildings. (Picture below).
![Outpost-12](https://user-images.githubusercontent.com/78278253/107552918-ba9dbc80-6b99-11eb-97bc-58c5009b406d.jpg)

Outpost_14 has also now been "added" in the previous location, it does NOT have a radiotower anymore, but does have an HQ building along with some other garrisonable structures and a functional helipad (picture below)
![outpost_14](https://user-images.githubusercontent.com/78278253/107553251-2c760600-6b9a-11eb-83a2-1119b8d448f4.jpg)

Both outposts have been tested in a LAN server and both spawn the initial garrison and counterattack correctly.
I do not suspect that any further major modifications should have to be made to the outposts.

Bob-Murphy reviewed (changes requested):
Looks good besides the black circle marker you have placed on the map for some reason.
Remove them and it should be good to go.
Also playtested the outposts and everything is working.

Preen1 commented:
No major changes in this commit. Just removed 4 circle markers that I used for referencing outpost_12.

Lazejun commented on Map-Templates/Antistasi-Malden.Malden/mission.sqm:
```diff
@@ -196,23 +200,45 @@ class AddonsMetaData
 		month=6;
 		hour=10;
 		minute=0;
+		startFogDecay=0.0049333;
```
😠


You still have to remove the Wind, Waves, Fog stuff, Scenario Data,
And add this one again
briefingName=$STR_antistasi_mission_info_malden_mapname_text;

Bob-Murphy replied:
I'll do that tomorrow and merged it is it's not done by then.
No worries, I already had my eyes on it.

Lazejun commented:
Is there any reason that there is one "Land_Wall_IndCnc_4_F" with Dynamic simulation while the rest doesen't have it?
Its the only one and you copy pasted it into both Outposts.

Preen1 commented:
No, Its likely something I did by accident while adjusting the outposts. Am removing it now

Preen1 commented:
Removed all the Fog, Wind, and scenario data as Laze pointed out. Also removed the custom attributes set for the 2 "Land_Wall_IndCnc_4_F" assets.

Lazejun reviewed (approved):
Looks good for now
